toolkits/landcover/impl/bands.js

```diff
@@ -146,9 +148,42 @@ function addFractionalYearBand(collection) {
   });
 }
 
+
+/**
+ * Compute a matrix multiply between the given image and a set of coefficients
+ * such as an eigenvector rotation or a Tasseled Cap transformation.
```

Comment:
Probably not worth it here.  I could take out the last clause of that sentence if you think it's a problem, but really, that's probably all this will ever get used for.

Reply:
I see you document it elsewhere, so I think it's ok.